### DISCUSSION TITLE
Who's using JK BMS - how did you wire it?

### DISCUSSION THREAD

**Discussion by greymda** [hoylabs/OpenDTU-OnBattery#875] [Q&A] [answered]
Hey community, would appreciate your help in how did you wire your JK BMS to the ESP32?
The wiki, to be honest, is a bit confusing for me, so please - bear with me.

A photo and model of the BMS would be awesome.

Happy weekend!

**Comment by DonJohnLong** [comment, marked as answer]
Here you go
![20240414_094054](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/130904315/89a6050c-5a46-45a3-977e-97f872b2625a)
![20240414_094208](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/130904315/fc6f2596-d422-4e7f-b638-47ff0ab9bb4d)
![20240414_094258](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/130904315/f5df3c39-0db0-4496-8d21-478c33d7081b)
![Screenshot_20240414_094405_Chrome](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/130904315/2abebd59-7d68-4275-b042-ccdafda24a9a)
![Screenshot_20240414_094429_Chrome](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/130904315/491ab8dc-7f54-4ce3-a78a-ecfb0d18f31c)

**Reply by greymda** [reply]
@DonJohnLong thank you! so you’ve used the direct connection between BMS and ESP32 boards? what’s the wire length? thanks again for the pictures!

**Reply by DonJohnLong** [reply]
Around 10-15cm, no problems at all

**Reply by greymda** [reply]
@DonJohnLong thank you!
